apps/test/integration/util/testCollectionUtils.js

```diff
@@ -1,17 +1,43 @@
+/* global LEVEL_TYPE */
+
 var _ = require('lodash');
 
+/**
+ * Given a list of levelSolutions subdirectories, returns only those matching
+ * the pattern specified in LEVEL_TYPE. If specified, LEVEL_TYPE must be contain
+ * one or more subdirectory names joined with '|' e.g. 'applab' or
+ * 'maze|artist|craft'.
```

Comment:
I wonder if at some point we should provide a good way to get instructions/information about all of our different test options (i.e. something like `npm run test:help`. At the least, should w ebe documenting LEVEL_TYPE in our readme somewhere?